dev-python/sqlglot/sqlglot-17.8.4.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..11} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="An easily customizable SQL parser and transpiler"
+HOMEPAGE="
+	https://sqlglot.com
+	https://github.com/tobymao/sqlglot/
+	https://pypi.org/project/sqlglot/
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# Tests require pyspark which then requires spark and so on,
```

Comment:
Can't you skip the tests needing pyspark?

Reply:
I falsely thought that the dependencies were integral to the tests. So yes, I can skip the the tests requiring duckdb and pyspark